invoices_app/views.py

```diff
@@ -337,7 +338,7 @@ def change_invoice_status(request, pk):
 
     status = request.POST.get('status')
 
-    if status not in [status for status, _ in INVOICE_STATUS]:
+    if status not in INVOICE_STATUSES_DICT.keys():
```

Comment:
NIT: I think that by default it checks the keys, so if you do `if status not in INVOICE_STATUSES_DICT` it will work